packages/react/src/components/Tile/Tile.js

```diff
@@ -485,15 +489,24 @@ export class ExpandableTile extends Component {
         : this.state.tileMaxHeight + this.state.tilePadding,
     };
     const content = this.getChildren().map((child, index) => {
-      return React.cloneElement(child, { ref: index });
+      // Changes aria-hidden above/below the fold content based on the state of expanded so the content doesn't get announced by screenreaders at the wrong time.
+      const isHidden = index === 0 ? expanded : !expanded;
```

Comment:
Could you update the comment to talk about how this updates over time? Something to the effect of when the tile is first rendered we want this announced, when it is expanded we want this announced, etc. and why `aria-hidden` is used to accomplish that.